api/app/events/repository.py

```diff
@@ -72,3 +72,11 @@ def get_event_fees(event_id, event_fee_ids):
                 EventFee.is_active==True)
             .all()
         )
+
+    @staticmethod
+    def get_all_admin(event_id):
+        return (
+            db.session.query(EventRole).get(event_id)
```

Comment:
When you use the `get` method it will query that tables primary key (in this case `EventRole.id`). It seems the intention here is to rather find all the admin `EventRoles` for a specific `event_id`. If that's the case, then the query should be something like
```python
def get_all_admin(event_id):
    return (
        db.session.query(EventRole)
        .filter_by(event_id=event_id, role='admin')\
        .all()
    )
```

Reply:
Actually I realised we can make this a bit better. Instead of querying for the `EventRole` and accessing the `user` attribute on it, let's just query for `AppUsers` directly. So on `/users/repository.py` you can do something like this. Then when sending the confirmation email, you'll just use the objects directly.

```python
def get_event_admins(event_id):
    return (
        db.session.query(AppUser)
        .join(EventRole, EventRole.user_id == AppUser.id)
        .filter_by(event_id=event_id)
        .all()
    )
```
Then